release-notes/Cabal-3.10.2.0.md

```diff
@@ -0,0 +1,32 @@
+Cabal and Cabal-syntax 3.10.2.0 changelog and release notes
+---
+
```

Comment:
Since these are also release notes, it won't hurt to say "Release 3.10.2.0 is strictly a bug-fix release, with the fixes listed below". Or something like that. The same at the top of the other file.